provider/source/src/properties/bin_cp_set.rs

```diff
@@ -64,78 +78,216 @@ macro_rules! expand {
-    (PropertyBinaryFullCompositionExclusionV1, "Comp_Ex"),
-    (PropertyBinaryChangesWhenCasefoldedV1, "CWCF"),
-    (PropertyBinaryChangesWhenCasemappedV1, "CWCM"),
-    (PropertyBinaryChangesWhenNfkcCasefoldedV1, "CWKCF"),
-    (PropertyBinaryChangesWhenLowercasedV1, "CWL"),
-    (PropertyBinaryChangesWhenTitlecasedV1, "CWT"),
-    (PropertyBinaryChangesWhenUppercasedV1, "CWU"),
-    (PropertyBinaryDashV1, "Dash"),
-    (PropertyBinaryDeprecatedV1, "Dep"),
-    (PropertyBinaryDefaultIgnorableCodePointV1, "DI"),
-    (PropertyBinaryDiacriticV1, "Dia"),
-    (PropertyBinaryEmojiModifierBaseV1, "EBase"),
-    (PropertyBinaryEmojiComponentV1, "EComp"),
-    (PropertyBinaryEmojiModifierV1, "EMod"),
-    (PropertyBinaryEmojiV1, "Emoji"),
-    (PropertyBinaryEmojiPresentationV1, "EPres"),
-    (PropertyBinaryExtenderV1, "Ext"),
-    (PropertyBinaryExtendedPictographicV1, "ExtPict"),
-    (PropertyBinaryGraphV1, "graph"),
-    (PropertyBinaryGraphemeBaseV1, "Gr_Base"),
-    (PropertyBinaryGraphemeExtendV1, "Gr_Ext"),
-    (PropertyBinaryGraphemeLinkV1, "Gr_Link"),
-    (PropertyBinaryHexDigitV1, "Hex"),
-    (PropertyBinaryHyphenV1, "Hyphen"),
-    (
-        PropertyBinaryIdCompatMathContinueV1,
-        "ID_Compat_Math_Continue"
-    ),
-    (PropertyBinaryIdCompatMathStartV1, "ID_Compat_Math_Start"),
-    (PropertyBinaryIdContinueV1, "IDC"),
-    (PropertyBinaryIdeographicV1, "Ideo"),
-    (PropertyBinaryIdStartV1, "IDS"),
-    (PropertyBinaryIdsBinaryOperatorV1, "IDSB"),
-    (PropertyBinaryIdsTrinaryOperatorV1, "IDST"),
-    (PropertyBinaryIdsUnaryOperatorV1, "IDSU"),
-    (PropertyBinaryJoinControlV1, "Join_C"),
-    (PropertyBinaryLogicalOrderExceptionV1, "LOE"),
-    (PropertyBinaryLowercaseV1, "Lower"),
-    (PropertyBinaryMathV1, "Math"),
-    (PropertyBinaryModifierCombiningMarkV1, "MCM"),
-    (PropertyBinaryNoncharacterCodePointV1, "NChar"),
-    (PropertyBinaryNfcInertV1, "nfcinert"),
-    (PropertyBinaryNfdInertV1, "nfdinert"),
-    (PropertyBinaryNfkcInertV1, "nfkcinert"),
-    (PropertyBinaryNfkdInertV1, "nfkdinert"),
-    (PropertyBinaryPatternSyntaxV1, "Pat_Syn"),
-    (PropertyBinaryPatternWhiteSpaceV1, "Pat_WS"),
-    (PropertyBinaryPrependedConcatenationMarkV1, "PCM"),
-    (PropertyBinaryPrintV1, "print"),
-    (PropertyBinaryQuotationMarkV1, "QMark"),
-    (PropertyBinaryRadicalV1, "Radical"),
-    (PropertyBinaryRegionalIndicatorV1, "RI"),
-    (PropertyBinarySoftDottedV1, "SD"),
-    (PropertyBinarySegmentStarterV1, "segstart"),
-    (PropertyBinaryCaseSensitiveV1, "Sensitive"),
-    (PropertyBinarySentenceTerminalV1, "STerm"),
-    (PropertyBinaryTerminalPunctuationV1, "Term"),
-    (PropertyBinaryUnifiedIdeographV1, "UIdeo"),
-    (PropertyBinaryUppercaseV1, "Upper"),
-    (PropertyBinaryVariationSelectorV1, "VS"),
-    (PropertyBinaryWhiteSpaceV1, "WSpace"),
-    (PropertyBinaryXdigitV1, "xdigit"),
-    (PropertyBinaryXidContinueV1, "XIDC"),
-    (PropertyBinaryXidStartV1, "XIDS")
+    (
```

Comment:
suggestion: can we rustfmt::skip this to avoid wrapping?

alternatively perhaps the syntax of `expand` can be turned into something like `PropType => PropMarker`. Or we can have it imply `icu::properties::props::`. Or something.